lib/l10n/arb/app_en.arb

```diff
@@ -0,0 +1,82 @@
+"enterNewEmail":"Enter new email",
+"enterRegisteredEmail":"Enter Registered Email",
+"enterCurrentPassword":"Enter current password",
+"resetPassword":"Reset password",
+"autoSync":"Auto sync",
+"alreadyHaveAnAccount":"Already have an account?",
+"syncNow":"Sync now",
+"availablePlatformsForSync":"Available platforms for sync",
+"automaticallySyncNotesWithCloud":"Automatically sync notes with cloud",
+"deletionFailed":"Deletion failed",
+"enableFingerPrintLogin":"Enable fingerprint login",
+"fingerPrintAthShouldBeEnabledInDeviceSettings":"Fingerprint auth should be enabled in device settings",
+"fingerprintLoginFailed":"Fingerprint login failed",
+"tooManyWrongAttempts":"Too many wrong attempts, please login with password",
+"emailUpdatedSuccessfully":"Email updated successfully, please login again",
+"passwordResetMailSent":"Password reset email sent",
+"incorrectPassword":"Incorrect password",
+"passwordVerified":"Password verified",
+"passwordsDontMatch":"Passwords don't match",
+"passwordResetSuccessful":"Password reset successful",
+"unexpectedErrorOccured":"Unexpected error occured",
+"pleaseSetupYourAccountToUseThisFeature":"Please setup your account to use this feature",
+"accountSetupSuccessful":"Account setup successful",
+"failedToFetchNote":"Failed to fetch note",
+"failedToSaveNote":"Failed to save note",
+"noteSavedSuccessfully":"Note saved successfully",
+"noteUpdatedSuccessfully":"Note updated successfully",
+"notesSyncSuccessfull":"Notes sync successful",
+"loginToEnableAutoSync":"Please login to enable auto-sync",
+"enableAutoSave":"Enable auto save",
+"dateFilter":"Date Filter",
+"from":"From",
+"to":"To",
+"areYouSureAboutLoggingOut":"Are you sure about logging out?",
+"sendFeedback":"Send feedback",
+"shareWithFriends":"Share with Friends",
+"chooseTheme":"Choose Theme",
+"appVersion":"App version",
+"continueAsGues":"Continue as guest",
+"byContinuingYouAgree":"By continuing, you agree to our",
+"privacyPolicy":"Privacy Policy",
+"closeTheApp":"Close the App?",
+"changePassword":"Change password",
+"changeEmail":"Change Email",
+"setupYourAccount":"Setup your Account",
+"logIn":"Log In",
+"logOut":"Log out",
+"signIn":"Sign In",
+"forgotPassword":"Forgot password",
+"newPassword":"New Password",
+"confirmNewPassword":"Confirm new password",
+"dontHaveAccount":"Don't have an account?",
+"signUp":"Sign Up",
+"tapToExpandTitle":"Tap here to expand title",
+"gallery":"Gallery",
+"link":"Link",
+"camera":"Camera",
+"video":"Video",
+"youHaveUnsavedChanges":"You have unsaved changes",
+"dropbox":"Dropbox",
+"signedInAs":"Signed in as",
+"lastSynced":"Last synced: ",
+"notAvailable":"Not available",
+"googleDrive":"Google Drive",
+"chooseTheSyncSource":"Choose the Sync Source",
+"done":"Done",
+"delete":"Delete",
+"cancel":"Cancel",
+"submit":"Submit",
+"appDescription":"Discover diaryVault - a diary app designed to help you capture your thoughts, memories, and moments effortlessly. Available now on the Play Store!"
+
```

Comment:
Format this file like a json file
```
{
  "language":"English",
  "@language":{
    "description":"The current Language"
  },
  
  "closeIcon":"close icon",
  "searchIcon":"search icon",
  "pageNotFound":"Page not found",
  "settings":"Settings",
  "enterNewEmail":"Enter new email"
  ...
```

Reply:
Didn't get you? It is formatted as a json file already

Reply:
Add one level of indentation for its contents
<img width="427" alt="image" src="https://github.com/SankethBK/diaryvault/assets/51091231/92cd2da5-972a-48c1-b619-ffa80898db5a">